Fail to run the server for new version of node

I'm running node version `v0.12.2` and when I tried to npm install it will fail on 

```
[09:29:26] Using gulpfile ~/projects/build.webmaker.org/Gulpfile.js
[09:29:26] Starting 'lint-app'...
[09:29:27] Starting 'bundle-app'...
[09:29:27] Starting 'less-app'...
/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/index.js:68
    }).done(undefined, cb);
       ^
TypeError: undefined is not a function
    at DestroyableTransform._transform (/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/index.js:68:8)
    at DestroyableTransform.Transform._read (/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at writeOrBuffer (/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:227:5)
    at DestroyableTransform.Writable.write (/Users/ali/projects/build.webmaker.org/node_modules/gulp-less/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:194:11)
    at write (/Users/ali/projects/build.webmaker.org/node_modules/gulp-sourcemaps/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:623:24)
    at flow (/Users/ali/projects/build.webmaker.org/node_modules/gulp-sourcemaps/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:632:7)
    at DestroyableTransform.pipeOnReadable (/Users/ali/projects/build.webmaker.org/node_modules/gulp-sourcemaps/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:664:5)
    at DestroyableTransform.emit (events.js:129:20)

npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.5
npm ERR! code ELIFECYCLE
npm ERR! webmaker-build@0.1.1 postinstall: `gulp build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webmaker-build@0.1.1 postinstall script 'gulp build'.
npm ERR! This is most likely a problem with the webmaker-build package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp build
npm ERR! You can get their info via:
npm ERR!     npm owner ls webmaker-build
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ali/projects/build.webmaker.org/npm-debug.log
☁  build.webmaker.org [github] 

```

I've tested with newer version of `gulp-less` and it works.

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "gulp-concat": "^2.4.3",
     "gulp-jshint": "git://github.com/spenceralger/gulp-jshint#overrideJshint",
     "gulp-jsxcs": "^0.1.7",
-    "gulp-less": "^2.0.1",
+    "gulp-less": "^3.0.3",
     "gulp-plumber": "^0.6.6",
     "gulp-sourcemaps": "^1.3.0",
     "gulp-uglify": "^1.0.2",

